### PR TITLE
GitHub Repository doc updates + NuGet.config uniformity

### PR DESCRIPTION
**Pull request by cheenamalhotra**
This PR updates documentation to add missing files and clarifies Contributor Guidelines.

> Note: There is work needed to format code in repository and follow the rules being added here, while that goes on in next PRs, this PR is only for adding documentation and gather feedback.

- Also updated NuGet.config usage from src directory, and removed duplicate references.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Please do:
+- **DO** report each issue as a new issue (but check first if it's already been reported)
+- **DO** respect Issue Templates and provide detailed information. It will make the process to reproduce the issue and provide a fix faster.
+- **DO** provide a minimal repro app demonstrating the problem in isolation will greatly speed up the process of identifying and fixing problems.
+- **DO** follow our [coding style](coding-style.md) (C# code-specific) when working on a Pull Request.
+- **DO** give priority to the current style of the project or file you're changing even if it diverges from the general guidelines.
+- **DO** consider cross-platform compatibility and supportability for all supported SQL and Azure Servers and client configurations.
+- **DO** include tests when adding new features. When fixing bugs, start with adding a test that highlights how the current behavior is broken.
+- **DO** consider concurrency when writing tests. Out pipelines run builds and tests in parallel using the same client and server configurations (in an isolated mode). E.g. Consider using dynamic table/database object names instead of hardcoded values (Use existing tests for reference).
```
Comment (Wraith2) [inline CONTRIBUTING.md]: Out->our

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- **DO** keep the discussions focused. When a new or related topic comes up it's often better to create new issue than to side track the discussion.
+- **DO** blog and tweet (or whatever) about your contributions, frequently!
+
+Please do not:
+
+- **DON'T** make PRs for style changes.
+- **DON'T** leave any artifacts on server in tests or leave open resources. Cleaning up all objects is highly appreciated to maintain test server health.
+- **DON'T** skip tests or run them conditionally unless necessary. If conditions are not met, test coverage will not be 100%. Use only pre-defined conditions that are already being run in pipelines.
+- **DON'T** surprise us with big pull requests. Instead, file an issue and start a discussion so we can agree on a direction before you invest a large amount of time.
+- **DON'T** commit code that you didn't write. If you find code that you think is a good fit to add to .NET Core, file an issue and start a discussion before proceeding.
```
Comment (Wraith2) [inline CONTRIBUTING.md]: Should this still say NET Core or change to SqlClient?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- **DO** provide a minimal repro app demonstrating the problem in isolation will greatly speed up the process of identifying and fixing problems.
+- **DO** follow our [coding style](coding-style.md) (C# code-specific) when working on a Pull Request.
+- **DO** give priority to the current style of the project or file you're changing even if it diverges from the general guidelines.
+- **DO** consider cross-platform compatibility and supportability for all supported SQL and Azure Servers and client configurations.
+- **DO** include tests when adding new features. When fixing bugs, start with adding a test that highlights how the current behavior is broken.
+- **DO** consider concurrency when writing tests. Out pipelines run builds and tests in parallel using the same client and server configurations (in an isolated mode). E.g. Consider using dynamic table/database object names instead of hardcoded values (Use existing tests for reference).
+- **DO** keep the discussions focused. When a new or related topic comes up it's often better to create new issue than to side track the discussion.
+- **DO** blog and tweet (or whatever) about your contributions, frequently!
```
Comment (Wraith2) [inline CONTRIBUTING.md]: Why is this suggested? I don't have a blog or twitter or anything like that.

Comment (cheenamalhotra) [reply CONTRIBUTING.md]: I brought in some stuff from corefx repro, it came with it :)
Feel free to ignore if it's not applicable to you. I don't see any harm in promoting contributions!

Comment (Wraith2) [reply CONTRIBUTING.md]: No, it just seems a bit weird and I don't remember seeing it in the corefx version. It isn't a problem.

**Pull request merged**